wizeline/project-lab
# Feat - #200 - Reorder Project Stages

#### What does this PR do?
In the "Edit project" view -> Contributor Path tab, adds the functionality to allow the users to drag and drop project stages to re-order them as they wish. Project Stages are collapsable now as well.

#### Where should the reviewer start?

- app/projects/components/ProjectContributorsPathForm/index.tsx
- app/projects/components/ProjectContributorsPathForm/StageCollapsableHeader.tsx
- app/projects/components/DragDropContainer.tsx

#### How should this be manually tested?

1. Run project locally
2. Log in
3. In Home view, select a project
4. (If not a member) join the project
5. Go to Edit view
6. Navigate to Contributor Path tab
7. Initially it should render a list of "Stage headers" (click on Stage header to display form)
8. Drag and drop whichever "Stage header"
9. Click "Update Stages"
10. You should be re-directed and see reordered stages

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?

[#200](https://github.com/wizeline/project-lab/issues/200) 

#### Screenshots

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/97301313/155213903-6dfab09b-56f9-41c8-a754-d1d2a14e9646.png">

#### Questions

N/A

#### Checklist
